docs/source/conf.py

```diff
@@ -632,6 +632,78 @@ def create_return_admonitions() -> dict[str, str]:
 RETURN_ADMONITION_FOR_CLASS_NAME = create_return_admonitions()
 
 
+def create_use_in_admonitions() -> dict[str, str]:
+    """Creates 'Use in' admonitions for classes whose instances that are accepted as arguments for
+    Bot's methods.
+    """
+
+    # First, generate a mapping of class names to the methods which accept them as arguments,
+    # i.e. {InlineQueryResult: [answer_inline_query, ...]}
+    methods_for_class_name = defaultdict(set)  # using set here because there can be repetitions
+
+    forward_ref_pattern = re.compile(r"ForwardRef\('(?P<class_name>.+?)'\)")  # non-greedy
+    skip_pattern = re.compile(
+        r"^typing.Union\[ForwardRef\('DefaultValue\[DVType]'\), ((float)|(bool)|(int)|(str))"
+        r"(, NoneType)?]$"
+    )
+
+    for method in [
+        m[0]
+        for m in inspect.getmembers(telegram.Bot, inspect.isfunction)  # not .ismethod
+        if not m[0].startswith("_")
+        and m[0].islower()  # islower() to avoid camelCase methods
+        and m[0] in telegram.Bot.__dict__  # method is not inherited from TelegramObject
+    ]:
+
+        sig = inspect.signature(getattr(telegram.Bot, method))
```

Comment:
maybe even including the signatures?

Reply:
I don't think that'll work in the current version because I need both the method name and its signature:

```py
            for method in relevant_methods_for_class:
                method_link = self._generate_link_to_method(method, klass)
                sig = inspect.signature(getattr(klass, method))
```